doc/source/api/proposed_schema_changes.md

```diff
@@ -0,0 +1,449 @@
+   which is implicated in the pathogenesis of several cancer types. I
+   could submit a query to ``/genotypephenotype/search`` with GIST as
+   the phenotype, *KIT* as the feature, and
+   `clinical study evidence <http://purl.obolibrary.org/obo/ECO_0000180>`__
+   as the evidence.
+
+In response, I will receive back a list of associations involving GIST
+and *KIT*, which I can filter for instances where imatinib is mentioned.
+URI's in the ``associations`` field could - hypothetically - be followed
+to discover that `GIST patients with wild-type *KIT* have decreased
+sensitivity to therapy with
+imatinib <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2651076/>`__.
+
+If I left both the ``genotype`` and ``evidence`` fields as ``null``, I
+would receive back all associations which involve GIST as a phenotype.
+
+2) As a non-Hodgkin's lymphoma researcher, I may know that the gene
+   *CD20* has abnormal expression in
+   `Hodgkin's lymphoma <http://purl.obolibrary.org/obo/DOID_8567>`__.
+   I might be interested in knowing whether *CD20* also has abnormal
+   expression in
+   `non-Hodgkin lymphoma <http://purl.obolibrary.org/obo/DOID_0060060>`__.
+   Therefore I could perform a query with *CD20* as a feature,
+   non-Hodgkin's lymphoma as a phenotype, and
+   `RNA sequencing <http://purl.obolibrary.org/obo/OBI_0001177>`__
+   as the evidence type.
+
+3) As a genetic counselor, I may be wondering if a mutation in one of my
+   clients' genes has ever been associated with a disease. I could then
+   do a query based on the gene name as the feature and
+   `disease <http://purl.obolibrary.org/obo/DOID_4>`__ as the
+   phenotype.
+
+For specifics of the json representations, please see the
+`server <https://github.com/ga4gh/server>`__ and
+`compliance <https://github.com/ga4gh/compliance>`__ repositories.
+
+Ontologies
+----------
+
+**Usage:** Multiple ontology terms can be supplied e.g. to describe a series
+of phenotypes for a specific sample. The ontology.avdl is not intended
+to model relationships between terms, or to provide mappings between
+ontologies for the same concept. Should an OntologyTerm be unavailable,
+or terms unmapped then an 'annotation' can be provided which can later
+be mapped to an ontology term using a service designed for this. Using
+OntologyTerm is preferred to using Annotation. Though annotations can be
+supplied with related ontology terms if desired. A use case could be
+when a free text annotation is very specific and a more general
+OntologyTerm is supplied.
+
+
+**Definitions:**
+
+*Annotation* - A free text annotation which is not an
+ontology term describing some attribute. Annotations have associations
+with OntologyTerms to allow these to be added after annotations are
+captured. OntologyTerms are preferred over Annotations in all cases.
+Annotations can be used in conjucntion with OntologyTerms
+
+*OntologyTerm* - the preferred term for the class in question. For example
```

Comment:
It would be better to refer to the metadata documentation here - this is already out of date.